app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int):
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+    result = {}
+    for person in people:
+        name = person["name"]
+        age = person["age"]
+        person = Person(name, age)
+        persons.append(person)
+    for item in people:
+        person = Person.people[item["name"]]
```

Comment:
The loop creates `Person` instances again for each person, which is unnecessary since they are already created in the previous loop. You can use the existing `Person` instances from the `persons` list instead of creating new ones.